app/src/main/java/org/wikiedufoundation/wikiedudashboard/ui/coursedetail/uploads/view/CourseUploadsFragment.kt

```diff
@@ -86,11 +86,10 @@ class CourseUploadsFragment : Fragment() {
         })
     }
 
-    private fun initializeToaster() {
-        courseUploadsViewModel.showMsg.observe(this, androidx.lifecycle.Observer {
-            val message = it.showMsg
-            context?.showToast(message)
-        })
+
+    override fun showMessage(message: String) {
+        view?.showSnackbar(message)
+
     }
```

Comment:
don't remove the initialiseToaster method.

Just remove
'context?.showToast(message)'
and instead use
'view?.showSnackbar(message)'

Also, initializeToaster will be renamed to initialiseSnackbar